packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -260,6 +262,22 @@ class CupertinoSliverNavigationBar extends StatelessWidget {
   }
 }
 
+// DELETE
```

Comment:
??

Reply:
I left this here so everything builds but this PR depends on https://github.com/flutter/engine/pull/4252